modules/promotion/src/Element/CouponRedemptionForm.php

```diff
@@ -100,12 +110,71 @@ public static function processForm(array $element, FormStateInterface $form_stat
       '#limit_validation_errors' => [
         array_merge($element['#parents'], ['code']),
       ],
+      '#submit' => [
+        [get_called_class(), 'elementButtonSubmit'],
```

Comment:
Why not call elementButtonSubmit and elementButtonRemove closer to what they do? applyCoupon and removeCoupon?